### ISSUE TITLE
Quadric Edge Collapse on large mesh

### ISSUE THREAD

**Issue by arountre** [cnr-isti-vclab/meshlab#152]
When attempting to perform Quadric Edge Collapse Decimation on a model of 20 million faces (default settings), the process stops at 90% completion when using Meshlab 2016.12.  We do not encounter this problem with the same model when using Meshlab v1.3.3-64.  Models with less than 10 million faces do not cause the problem.  Workstation has 128GB RAM, and we are not approaching that limit.

**Comment by granzuglia** [comment]
Using Win10 on 32GB RAM laptop I was able to simplify a 60 million triangles model to a 6 million one. I compiled the current MeshLab version available from GitHub server. It was quite a bit long (something around 10 minutes, maybe) but that's quite normal. I tried also with the default parameters (getting a model of 30 million triangles) and I was successful too. Can I know the specs of your GPU? Did you follow the MeshLab 2016.12 setup instructions (amount of gpu ram specifically dedicated to MeshLab) contained in this video https://www.youtube.com/watch?v=ZbyzQHo666E ?
Let me know if you have still the problem.
